### DISCUSSION TITLE
Keybinding to search history for currently-entered string

### DISCUSSION THREAD

**Discussion by singpolyma** [xonsh/xonsh#4485] [Q&A]
I know xonsh almost does this by default, with the ghost-auto-complete, but I have a strong muscle memory that Ctrl+N should take whatever I've already typed and search history for it.  .inputrc equivalent `Control-n: history-search-backward` any way to get this in xonsh?

**Comment by jnoortheen** [comment]
Check out fzf-widgets - https://github.com/laloch/xontrib-fzf-widgets, and binding the necessary history search to ctrl+N

**Reply by singpolyma** [reply]
Hmm, that seems to be an interactive search.  I just want to search in history for whatever I have already typed on the command line, not open a search to type new things into.
